schema/bom-1.5.schema.json

```diff
@@ -137,11 +137,39 @@
           "description": "The date and time (timestamp) when the BOM was created."
         },
         "tools": {
-          "type": "array",
-          "title": "Creation Tools",
-          "description": "The tool(s) used in the creation of the BOM.",
-          "additionalItems": false,
-          "items": {"$ref": "#/definitions/tool"}
+          "oneOf": [
+            {
+              "type": "object",
+              "title": "Creation Tools",
+              "description": "The tool(s) used in the creation of the BOM.",
+              "additionalProperties": false,
+              "properties": {
+                "components": {
+                  "type": "array",
+                  "additionalItems": false,
```

Comment:
What would be nice is to ask for relevant configuration, parameters, and environment variables used by the component or the service to generate the SBoM. Inspired by commandLine and arguments in sarif.

https://docs.oasis-open.org/sarif/sarif/v2.1.0/os/sarif-v2.1.0-os.html#_Toc34317570

Reply:
That's all part of formulation #31